controls/pcidss_4_ocp4.yml

```diff
@@ -1285,7 +1285,7 @@ controls:
           - base
         status: not applicable
         notes: |-
-          OpenShift doesn't manage wireless environments nor they security configurations.
```

Comment:
Aha - for a second I was wondering if I left a comment in the wrong place, but here you're just addressing a similar comment just elsewhere in the doc?

Reply:
@rhmdnd Fixed again